db/migrate/20200617014509_add_subscriber_email_index_to_user_subscriptions.rb

```diff
@@ -0,0 +1,45 @@
+class AddSubscriberEmailIndexToUserSubscriptions < ActiveRecord::Migration[6.0]
+  disable_ddl_transaction!
+
+  def up
+    if index_exists?(:user_subscriptions, %i[subscriber_id user_subscription_sourceable_id user_subscription_sourceable_type])
```

Comment:
Can't wait for when we update to Rails 6.1 and get access to `if_exists` and `if_not_exists` options on indexes 🎉